addons/account_peppol/wizard/service_wizard.py

```diff
@@ -0,0 +1,109 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+from markupsafe import Markup
+
+from odoo import api, fields, models, _
+
+_logger = logging.getLogger(__name__)
+
+
+class PeppolService(models.TransientModel):
+
+    _name = 'account_peppol.service'
+    _order = 'document_name'
+    _description = 'Peppol Service'
+
+    wizard_id = fields.Many2one('account_peppol.service.wizard')
+    document_identifier = fields.Char()
+    document_name = fields.Char()
+    enabled = fields.Boolean(default=False)
+
+
+class PeppolServiceConfig(models.TransientModel):
+
+    _name = 'account_peppol.service.wizard'
+    _description = 'Peppol Services Wizard'
+
+    edi_user_id = fields.Many2one('account_edi_proxy_client.user', string='EDI user')
+    service_json = fields.Json(help="JSON representation of peppol services as retrieved from the peppol server.")
+    service_info = fields.Html(compute='_compute_service_info')
+    service_ids = fields.One2many(
+        comodel_name='account_peppol.service',
+        inverse_name='wizard_id',
+        readonly=False,
+    )
+
+    # -------------------------------------------------------------------------
+    # COMPUTES
+    # -------------------------------------------------------------------------
+
+    def _compute_service_info(self):
+
+        for wizard in self:
+            message = ''
+            supported_doctypes = self.env['res.company']._peppol_supported_document_types()
+            if (non_configurable := [
+                identifier for identifier in (wizard.service_json or {})
+                if identifier not in supported_doctypes
+            ]):
+                message = Markup('%s<ul>%s</ul>') % (_(
+                    "The following services are listed on your participant but cannot be configured here. "
+                    "If you wish to configure them differently, please contact support."),
+                    Markup().join(Markup('<li>%s</li>') % (
+                        wizard.service_json[identifier]['document_name'])
```

Comment:
on the `if`, you added `wizard.service_json or {}` meaning `wizard.service_json` could be falsy. Be consistent plz: remove `or {}` or deal with the falsy value in both cases.

Reply:
This code is unreachable if `wizard.service_json` is falsey
It's essentially the same as doing an 
```python
if x in somedict:
   somedict[x]
```

To clarify, the reason for doing `(wizard.service_json or {})` everywhere is simply that
`{}` and `[]` are valid json, but the odoo json field is incapable of representing them (and instead represents them as `False`), which is annoying because ordinarily `json` content is iterable in one way or another

```
>>> wizard.service_json = []
>>> print(wizard.service_json)
False
>>> wizard.service_json = {}
>>> print(wizard.service_json)
False
```